test/orange/univ/detail/SenseCalculator.test.cc

```diff
@@ -109,26 +109,22 @@ using TestTypes = ::testing::
 
 TYPED_TEST_SUITE(SenseCalculatorTest, TestTypes, );
 
+#if CELERITAS_DEBUG
 TYPED_TEST(SenseCalculatorTest, one_volume)
 {
-    using MySenseCalc = TypeParam;
-    {
-        typename TestFixture::OneVolInput geo_inp;
-        this->build_geometry(geo_inp);
-    }
+    typename TestFixture::OneVolInput geo_inp;
+    this->build_geometry(geo_inp);
 
     // Test this degenerate case (no surfaces)
     OnFace face;
-    MySenseCalc calc_senses = this->construct_sense_calculator(
-        this->make_surf_visitor(),
-        this->make_volume_view(LocalVolumeId{0}),
-        Real3{123, 345, 567},
-        face);
-    if (CELERITAS_DEBUG)
-    {
-        EXPECT_THROW(calc_senses(FaceId{0}), DebugError);
-    }
+    EXPECT_THROW(this->construct_sense_calculator(
+                     this->make_surf_visitor(),
+                     this->make_volume_view(LocalVolumeId{0}),
```

Comment:
Oh yeah if we're taking these as const references perhaps it's better to construct them locally and pass it in as an lvalue?

Reply:
This should be legal since the lifetime of these references is extended to the end of the statement maybe clearer to pass lvalues